cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -222,11 +222,11 @@ instance FromJSON PartialNodeConfiguration where
              }
 
       parseHardForkProtocol v = do
-        npcTestShelleyHardForkAtEpoch   <- v .:? "TestShelleyHardForkAtEpoch"
+        npcShelleyHardForkAtEpoch   <- v .:? "ShelleyHardForkAtEpoch"
         npcTestShelleyHardForkAtVersion <- v .:? "TestShelleyHardForkAtVersion"
```

Comment:
Hmm, I'm not sure about the `Test-` prefix thing. `ShelleyHardForkAtEpoch` should *never* be used for mainnet, only for testing/testnets, because for mainnet we want to trigger the hard fork *at a version*. Following that reasoning, `TestShelleyHardForkAtVersion` should become `ShelleyHardForkAtVersion`, but that *is* used for mainnet.

Reply:
Ok, I will revert that part.